src/components/Hero/Hero.tsx

```diff
@@ -110,9 +109,12 @@ export default function Hero(props: React.PropsWithChildren<HeroProps>) {
 
     return (
         <div
-            className={bem(heroBaseClass, heroModifiers)}
+            className={bem(heroBaseClass, heroModifiers, blockName, [
+                className,
+            ])}
             data-responsive-background-image
             style={backgroundImageStyle}
+            {...attributes}
```

Comment:
(I think) this is an example of a div where this makes sense.  You aren't really allowed to pass whatever you want into <Hero>, it's supposed to generate the div for you, so if you want more functionality this is your only choice.

Reply:
Is this different from what `Modal` is doing? `Modal` also generates a wrapping div for you that you otherwise cannot add attributes to.

Reply:
I think the difference is that in `Modal` you're expected to pass your own content that you control.  In `Hero` you are supposed to pass 

```
<Hero
  backgroundImageSrc="https://placeimg.com/1600/800/arch"
  heading={<Heading blockName="hero" id="1" level={1} text="Hero Primary"/>}
  heroType="PRIMARY"
/>
```

Reply:
That means if you wanted to put a customProp, you could go 

`<Modal> <div customProp=customFunction /> </Modal>` 

but with Hero where would you put it?

Reply:
I see what you're saying with Modal—you can always pass in customized children, so you'd probably never need to use attributes. But isn't the whole point of attributes to acknowledge that we can't anticipate everyone's needs? By adding attributes to Modal, in the event that someone _does_ need to add an attribute to the Modal where `tabindex` is, they can. It also means everything is extensible in the same way more or less. It could be I'm being influenced by PatternLab's philosophy here (they universally add attributes to everything without making distinctions). Is there harm to doing it this way? (brb, gonna go read Edwin's comments.)

Reply:
I wonder if it might be better to only add attributes to components that can take handlers.  When working in DRB I have only found that I needed attributes in `<input>` or `<button>` or `<a>` 

Like, the extra className handles the most common case to put anything in `<div>` right?  

I say this because once we add `<div>`s stuff gets complicated?  
Let's go back to the Hero example - it compiles down to 

```
<div class="hero hero--primary" 
data-responsive-background-image="true" s
tyle="background-image: url(&quot;https://placeimg.com/1600/800/arch&quot;);"
{attributes will go here if this is merged as is}>
<div class="hero__content"
{But what if I want attributes here? where would I pass that?}>
<h1 class="hero__heading" id="1" 
{This is ok because the header is passed in as a separate element}>Hero Primary</h1>
</div>
</div>
```

Reply:
I see what you're saying-basically that if you're not passing attributes to the "lowercase" component, as we've been calling it, it isn't as easily identifiable where those attributes are getting added. By not including them on such components, we feel the API is more clear. Am I summing this up well?

Reply:
yup!  That's exactly it.  
I think to be clearest, let's only put attributes in `lowercase` components for now.  If people need them for divs, we will revisit once we have clear use cases.